clang/test/SemaCXX/warn-unused-private-field.cpp

```diff
@@ -20,6 +20,17 @@ class SpaceShipDefaultCompare {
   int operator<=>(const SpaceShipDefaultCompare &) const = default;
 };
 
+class UnusedConstPrivateField {
+ public:
+  UnusedConstPrivateField() : unused_(0) {}
+ private:
+  const int unused_; // expected-warning{{private field 'unused_' is not used}}
+};
```

Comment:
Its not relevant that the field is const, and the constructor is also irrelevant to the bug in question.

```suggestion
class HasUnusedField {
  int unused_; // expected-warning{{private field 'unused_' is not used}}
};
```